circle.yml

```diff
@@ -16,6 +16,11 @@ dependencies:
 
 test:
   override:
+    # if the line below produce an error in CircleCI,
+    # you have to update the slicer dependencies image in SlicerDocker
+    # In order to do it, please create a Pull Request like this one https://github.com/thewtex/SlicerDocker/pull/21
+    # This will build and push all images based on the newer version of Slicer
```

Comment:
Folks that receive the build error will not know that this comment is here.

Please try adding

```
 echo "Please create a Pull Request on https://github.com/thewtex/SlicerDocker according to the Update section of the project README.";
```

before the `false` instead.